src/Actor.cpp

```diff
@@ -1078,14 +1078,13 @@ void Actor::ScaleTo( const RectF &rect, StretchType st )
 	float fNewZoomY = std::abs(rect_height / m_size.y);
 
 	float fNewZoom = 0.f;
-	switch( st )
+	if (st == cover)
```

Comment:
For enums I actually do prefer switches. I think it would be worth to:
1) Prefix the cases with the enum type e.g. `StretchType::Cover`
2) Rename the enum with better casing. Style guide says to name it similar to constants (i.e UpperCamelCase with a `k` prefix, but I think for now we don't need the `k` prefix. So just `Cover` and `FitInside`. All lowercase is a bit confusing because I thought it was a variable name originally.